splunk_add_on_ucc_framework/commands/rest_builder/user_defined_rest_handlers.py

```diff
@@ -299,6 +338,20 @@ def add_definition(
         if definition.endpoint in self._endpoints:
             raise ValueError(f"Duplicate REST handler endpoint: {definition.endpoint}")
 
+        if (
+            definition.registerHandler is not None
+            and "file" in definition.registerHandler
+        ):
+            file = definition.registerHandler["file"]
+
+            if file.endswith(".py"):
+                file = file[:-3]
+
+            if file in self._files:
+                raise ValueError(f"Duplicate REST handler file: {file}")
```

Comment:
Can we mention action item for add-on developers if they've provided the same name, instead of this message?
Also, do we see a use case where two different endpoints can/may have the same logic, and hence they kept the same file, or is it disallowed by Splunk itself?